modules/OFConnectionManager2/module/src/ofconnectionmanager.c

```diff
@@ -976,10 +980,7 @@ ind_cxn_notify_closed(connection_t *cxn)
 static void
 controller_stop_cxn(connection_t *cxn)
 {
-    if (cxn == NULL) {
-        AIM_LOG_INTERNAL("Controller stop on null cxn, skipping");
-        return;
-    }
+    AIM_TRUE_OR_DIE(cxn != NULL, "NULL connection passed to controller_stop_cxn");
```

Comment:
i think i've seen this message show up before, probably when the controller is removed and there is no current connection to the controller.  maybe we should add a NULL check before calling controller_stop_cxn from indigo_controller_remove?